diff --git a/node_modules/amplify-docs/src/components/Menu/Directory/index.tsx b/node_modules/amplify-docs/src/components/Menu/Directory/index.tsx
index 65e4470..b91501e 100644
--- a/node_modules/amplify-docs/src/components/Menu/Directory/index.tsx
+++ b/node_modules/amplify-docs/src/components/Menu/Directory/index.tsx
@@ -57,7 +57,7 @@ class DirectoryGroup extends React.Component<
     ) {
       this.state = {isExpanded: true};
     } else {
-      this.state = {isExpanded: this.props.url.startsWith("/start")};
+      this.state = {isExpanded: true};
     }
   }
 
diff --git a/node_modules/amplify-docs/src/components/Menu/FilterSelect/index.tsx b/node_modules/amplify-docs/src/components/Menu/FilterSelect/index.tsx
index 61372fc..5decc7e 100644
--- a/node_modules/amplify-docs/src/components/Menu/FilterSelect/index.tsx
+++ b/node_modules/amplify-docs/src/components/Menu/FilterSelect/index.tsx
@@ -75,7 +75,14 @@ export default class FilterSelect extends React.Component<
     }
 
     return (
-      <Link href={href} key={name}>
+      <Link
+        href={{
+          pathname: this.props.url,
+          query: { platform: name },
+        }}
+        key={name}
+        scroll={false}
+      >
         <a onClick={this.toggleVis}>
           <img
             alt={filterMetadataByOption[name]?.label + " icon"}
diff --git a/node_modules/amplify-docs/src/components/Menu/RepoActions/index.tsx b/node_modules/amplify-docs/src/components/Menu/RepoActions/index.tsx
index 07a67f6..50acf3f 100644
--- a/node_modules/amplify-docs/src/components/Menu/RepoActions/index.tsx
+++ b/node_modules/amplify-docs/src/components/Menu/RepoActions/index.tsx
@@ -24,9 +24,9 @@ const getLabelForPath = (path) => {
 };
 
 function createIssueLink(directoryPath, url) {
-  url = `https://docs.amplify.aws${url}`;
+  url = `https://ui.docs.amplify.aws${url}`;
   const NEW_GITHUB_ISSUE_LINK =
-    "https://github.com/aws-amplify/docs/issues/new";
+    "https://github.com/aws-amplify/amplify-ui/issues/new";
   const params = [
     "title=[Feedback]FEEDBACK_TITLE_HERE",
     `labels=${encodeURIComponent(getLabelForPath(directoryPath))}`,
@@ -46,7 +46,7 @@ function createEditLink(directoryPath) {
     .split("/")
     .map(encodeURIComponent)
     .join("/");
-  return `https://github.com/aws-amplify/docs/edit/main/src/pages${safePath}.mdx`;
+  return `https://github.com/aws-amplify/amplify-ui/edit/main/docs/src/pages/${safePath}/index.page.mdx`;
 }
 
 export default function RepoActions({directoryPath, url}) {
diff --git a/node_modules/amplify-docs/src/components/Menu/index.tsx b/node_modules/amplify-docs/src/components/Menu/index.tsx
index a4b7028..f0cced1 100644
--- a/node_modules/amplify-docs/src/components/Menu/index.tsx
+++ b/node_modules/amplify-docs/src/components/Menu/index.tsx
@@ -72,7 +72,7 @@ export default class Menu extends React.Component<MenuProps, MenuState> {
         this.props.url.includes("/ui-legacy")) &&
       this.props.filterKey !== "react-native"
     ) {
-      showVersionSwitcher = true;
+      showVersionSwitcher = false;
     }
     if (this.state.isOpen) {
       return (
diff --git a/node_modules/amplify-docs/src/components/TableOfContents/index.tsx b/node_modules/amplify-docs/src/components/TableOfContents/index.tsx
index e900068..1cc233c 100644
--- a/node_modules/amplify-docs/src/components/TableOfContents/index.tsx
+++ b/node_modules/amplify-docs/src/components/TableOfContents/index.tsx
@@ -24,17 +24,18 @@ export default function TableOfContents({children, title}) {
   if (children.length === 0) {
     return <></>;
   }
-  window.onload = (_) => {
-    if (window.location.href.includes("#")) {
-      const hash = window.location.href.split("#")[1];
-      scroll(hash);
-    }
-  };
   let headers = [];
   let headerQueries = [];
   let activeLink = 0;
   let previousLink = 0;
+
   useEffect(() => {
+    window.onload = (_) => {
+      if (window.location.href.includes("#")) {
+        const hash = window.location.href.split("#")[1];
+        scroll(hash);
+      }
+    };
     const idSet = new Set();
     const headings = document.querySelectorAll("a > h2, a > h3");
     const headings2 = document.getElementById("toc").querySelectorAll("a");
@@ -55,11 +56,11 @@ export default function TableOfContents({children, title}) {
       }
       (headings[i].parentElement as HTMLAnchorElement).onclick = () => {
         setTimeout(scroll.bind(undefined, uniqueId), 50);
-        return false;
+        // return false;
       };
       headings2[i].onclick = () => {
         setTimeout(scroll.bind(undefined, uniqueId), 50);
-        return false;
+        // return false;
       };
     }
     headers = Array.from(headings).map((heading) => heading.id);
@@ -93,18 +94,18 @@ export default function TableOfContents({children, title}) {
                 aTag.classList.add("active");
               });
             if (activeElement) {
-              router.replace(
-                window.location.href.split("#")[0],
-                window.location.href.split("#")[0] + "#" + activeElement,
-                {shallow: true, scroll: false},
-              );
+              // router.replace(
+              //   window.location.href.split("#")[0],
+              //   window.location.href.split("#")[0] + "#" + activeElement,
+              //   {shallow: true, scroll: false},
+              // );
             }
           } else {
-            router.replace(
-              window.location.href.split("#")[0],
-              window.location.href.split("#")[0],
-              {shallow: true, scroll: false},
-            );
+            // router.replace(
+            //   window.location.href.split("#")[0],
+            //   window.location.href.split("#")[0],
+            //   {shallow: true, scroll: false},
+            // );
           }
         }
       }
@@ -113,7 +114,7 @@ export default function TableOfContents({children, title}) {
     return function cleanup() {
       document.removeEventListener("scroll", scrollHandler);
     };
-  }, []);
+  }, [children]);
 
   return (
     <TOCStyle id="toc">
diff --git a/node_modules/amplify-docs/src/pages/ChooseFilterPage.tsx b/node_modules/amplify-docs/src/pages/ChooseFilterPage.tsx
index 561b000..8dd99d0 100644
--- a/node_modules/amplify-docs/src/pages/ChooseFilterPage.tsx
+++ b/node_modules/amplify-docs/src/pages/ChooseFilterPage.tsx
@@ -79,7 +79,7 @@ function ChooseFilterPage({
           {filters.map((filter) => (
             <Card
               className="vertical"
-              href={`${url}/q/${filterKind}/${filter}`}
+              href={`${url}?platform=${filter}`}
               key={filter}
             >
               <CardGraphic
